third_party/txt/src/txt/paragraph.cc

```diff
@@ -1123,7 +1124,23 @@ std::vector<Paragraph::TextBox> Paragraph::GetRectsForRange(size_t start,
 
   std::vector<Paragraph::TextBox> boxes;
   for (const auto& kv : line_boxes) {
-    boxes.insert(boxes.end(), kv.second.begin(), kv.second.end());
+    if (rect_style & RectStyle::kTight) {
+      // Ignore line max height and width and generate tight bounds.
+      boxes.insert(boxes.end(), kv.second.begin(), kv.second.end());
+    } else {
+      // Set each box to the max height of each line to ensure continuity.
+      float min_top = DBL_MAX;
+      float max_bottom = -1;
```

Comment:
why -1?

Reply:
Any value smaller than zero should do, since the bottom of it cannot be above the top, and the highest the top can be is zero.

Reply:
why not zero, then?